doc/manual/manual.tex

```diff
@@ -2771,20 +2773,126 @@ \subsubsection{Compiling \aspect{} and generating documentation}
+\end{verbatim}
+
+\paragraph{Installing missing prerequisites:}
+
+As described in the last paragraph \texttt{candi} is able to install some of
+the prerequisites mentioned in Subsection \ref{subsubsec:system_prerequisites},
+if they are not available or packaged for your system. This can be especially
+useful on cluster systems. For a list of available packages see the
+\texttt{candi.cfg} file in the \texttt{candi} directory and uncomment the
+missing packages.
+
+\paragraph{Using custom compiler and linker flags:}
+
+If your system requires custom compiler or linker flags (e.g. \texttt{-lmpi} to
+link to the MPI libraries), you can set the common environment variables
+(\texttt{CFLAGS,CPPFLAGS,LDFLAGS}) in your terminal, or include the
+corresponding commands into the \texttt{candi.cfg} file in the \texttt{candi}
+directory to make sure they are set whenever \texttt{candi} is run.
+
+\paragraph{Installation using Intel's MKL as BLAS/LAPACK:}
+
+A common obstacle during installation is that many cluster systems utilize
+Intel's MKL library as an implementation of the BLAS/LAPACK routines.
+\texttt{candi} can automatically configure the packages that depend on this
+implementation, if it is told to do so. To this purpose modify the
+corresponding section of the \texttt{candi.cfg} file in the \texttt{candi}
+directory. It should contain the following lines:
+
 \begin{verbatim}
- -DBUILD_SHARED_LIBS=OFF
- -DTPL_FIND_SHARED_LIBS=OFF
+MKL=ON
+MKL_DIR=${MKLROOT}/lib/intel64
 \end{verbatim}
-\item \textit{\pfrst{}:} Change items "--enable-shared" to "--enable-static" in p4est-setup.sh lines 83 and 97.
-\item \textit{\dealii{}:} Add the following lines to your call to cmake:
+
+Note that the value of \texttt{MKL\_DIR} needs to point to your mkl
+implementation, i.e. the folder that contains the libraries (like
+``libmkl\_core.so'' and others). The value \texttt{\${MKLROOT}/lib/intel64} is
+simply a common place for the installation, and might be different on your
+system (consult you cluster's documentation or contact technical support).
+
+\paragraph{Installation without support for C++11:}
+
+The C++ programming language has received a major update in 2011 introducing
+the C++11 standard. Compilers compatible with this new version were released
+shortly afterwards and have been available for some time (e.g.
+GCC-4.8.0, the first GCC version considered to have full C++11 support, was
+released in 2013, other compatible compilers are Clang 3.3, and Intel 15 or
+newer).
+
+We are aware that some clusters without regular support might not provide C++11
+support and have therefore kept \aspect{} compatible to older compilers in line
+with the implementations of deal.II (see also
+\url{https://github.com/dealii/dealii/wiki/Frequently-Asked-Questions#does-dealii-require-c11-support}).
+Note however, that \trilinos{} and \dealii{} already dropped support for
+non-C++11 compilers in their newest releases and development version
```

Comment:
mention specific releases, or this paragraph will become outdated pretty quickly.